test/carbon/Strategies.ts

```diff
@@ -1427,6 +1427,23 @@ describe('Strategy', () => {
                 );
         });
 
+        it('should be able to delete a disabled order', async () => {
```

Comment:
A more accurate description is `should be able to delete a strategy when both orders are disabled`.
I would also explicitly create such strategy (I think that you are creating it implicitly here).

Perhaps even split this test into 3 tests:
- 1st order disabled, 2nd order enabled
- 1st order enabled, 2nd order disabled
- 1st order disabled, 2nd order disabled

Reply:
added